chapters/hpmor-chapter-064.tex

```diff
@@ -10,13 +10,14 @@
 \makeatother
 
 \newcommand{\OmakeIVsection}[2][1.5]{%
-\OmakeIVspecialsection[#1]{\MakeUppercase{#2}}}
+  \OmakeIVspecialsection[#1]{\MakeUppercase{#2}}}
 
 \chapter{Omake Files IV, Alternate Parallels}
 
 \OmakeIVspecialsection[1.6]{\fontspec[ExternalLocation]{RingBearer}
-\settowidth{\versewidth}{\mbox{the}} Lord\scalebox{.40}{\parbox[b]{\versewidth}{%
-\centering of\\\nointerlineskip\vskip 4pt the}}Ratîonalît\raisebox{-.32ex}{Y}}
+\settowidth{\versewidth}{\mbox{the}}
+Lord\scalebox{.40}{\parbox[b]{\versewidth}{%
+        \centering of\\\nointerlineskip\vskip 4pt the}}Ratîonalît\raisebox{-.32ex}{Y}}
```

Comment:
For PR #66?

Reply:
good point, looks I modified it in both, now copied there too